lte/gateway/python/integ_tests/s1aptests/test_guti_attach.py

```diff
@@ -0,0 +1,220 @@
+"""
```

Comment:
what is the reason for file name change?

Reply:
what is the difference from `test_guti_attach_with_zero_mtmsi.py`

Reply:
I do not see the connection of this test to the metrics naming.

Reply:
i think this file got included as part of a rebase with other commit I had on my local history, I will remove it, this is not supposed to be added